apps/explorer/lib/explorer/chain.ex

```diff
@@ -7860,10 +7894,28 @@ defmodule Explorer.Chain do
     |> Repo.one()
   end
 
-  @spec delete_celo_pending_epoch_operation(Hash.Full.t()) :: CeloPendingEpochOperation.t()
-  def delete_celo_pending_epoch_operation(block_hash) do
+  @spec falsify_or_delete_celo_pending_epoch_operation(
+          Hash.Full.t(),
+          :fetch_epoch_rewards | :fetch_validator_group_data
+        ) :: CeloPendingEpochOperation.t()
+  def falsify_or_delete_celo_pending_epoch_operation(block_hash, operation_type) do
```

Comment:
this isn't relevant to the chain. we can do this inside of the CeloPendingEpochOperation import runner.

Similar to how it is done with internal transactions https://github.com/celo-org/blockscout/blob/a801b8f2f5d8d5c4da27c37ceeb3bf26397d36f7/apps/explorer/lib/explorer/chain/import/runner/internal_transactions.ex#L39